packages/bezier-react/src/components/Tabs/Tabs.module.scss

```diff
@@ -66,6 +66,7 @@ $tab-item-indicator-height: 3px;
 
   /* stylelint-disable-next-line no-descending-specificity */
   &:where([data-state='inactive']) {
+    overflow: hidden;
```

Comment:
1. This is not necessary since already applied in `TabItems`.
2. `bottom` position of underline should be lifted by `$tab-item-indicator-height`. You can run storybook locally by `yarn dev` command and check how it looks.

Reply:
I tried removing overflow:hidden from TabItem and adding it back in TabItems but it seems to bring back the focus overflowing issue

Reply:
I think it's good 👍 

We manage versioning by [changeset](https://github.com/channel-io/bezier-react/blob/main/.github/CONTRIBUTING.md#add-a-changeset), so can you add it? patch version bump would be good. This is [example](https://github.com/channel-io/bezier-react/pull/2278/commits/61bded0d77163f9b90f6c5b70b3e1676d40963f9).

Reply:
> I think it's good 👍
> 
> We manage versioning by [changeset](https://github.com/channel-io/bezier-react/blob/main/.github/CONTRIBUTING.md#add-a-changeset), so can you add it? patch version bump would be good. This is [example](https://github.com/channel-io/bezier-react/pull/2278/commits/61bded0d77163f9b90f6c5b70b3e1676d40963f9).

I tried adding one. I hope it's fine.